packages/create-widget.template.react.v2/templates/index.html

```diff
@@ -6,6 +6,7 @@
     <title>Widget: Ontology SDK + React</title>
   </head>
   <body>
+    <div id="root"></div>
```

Comment:
This is a bit confusing but this index.html for the widgets don't actually get used for rendering, it's only used by the vite plugin to extract out the entrypoints. For rendering we provide the index.html but I updated these as well for consistency